airflow/contrib/operators/dataproc_operator.py

```diff
@@ -121,6 +121,9 @@ class DataprocClusterCreateOperator(DataprocOperationBaseOperator):
         config files (e.g. spark-defaults.conf), see
         https://cloud.google.com/dataproc/docs/reference/rest/v1/projects.regions.clusters#SoftwareConfig
     :type properties: dict
+    :param optional_components: List of optional cluster components, for more info see
+        https://cloud.google.com/dataproc/docs/reference/rest/v1/ClusterConfig#Component
+    :type optional_components: list[str]
```

Comment:
```suggestion
    :type optional_components: List[str]
```

Reply:
There is inconsistency in this file in the way types are written. We have to review all types then we will standarize it.  Now the current version is more popular.